src/Illuminate/Database/Query/Builder.php

```diff
@@ -1690,7 +1690,9 @@ public function max($column)
      */
     public function sum($column)
     {
-        return $this->aggregate(__FUNCTION__, [$column]);
+        $result = $this->aggregate(__FUNCTION__, [$column]);
+
+        return $result ?: 0;
```

Comment:
This was just done to revert it back to its original state before the reverted code was applied. If Taylor wants, I can do it like this.